### PR TITLE
Template changes and example of changes

### PR DESCRIPTION
**Pull request by Astarsys**
- updated description for APN settings
- for JSON - moved comments below it
- added examples
These calls as for example. Filled using the template

### CODE REVIEW

File: docs/backend-api/resources/tracking/tracker/alarm_mode.md [hunk 1]
```diff
@@ -15,28 +15,46 @@ Gets the state of alarm mode of device.
...
 
 ```json
 {
  "success": true,
- "enabled": {boolean} // true if alarm mode is enabled
+ "enabled": true
 }
 ```
 
+* enabled - true if alarm mode enabled
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/alarm_mode.md:51]: Missed a dot at the end. `enabled` should be escaped (as all fields and entity names).

File: docs/backend-api/resources/tracking/tracker/apn_settings.md [hunk 2]
```diff
@@ -5,23 +5,42 @@ description: APN settings by tracker ID
 
 API base path: `/tracker/apn_settings`
 
+APN is short of Access Point Name and provides a device with the information needed to connect to wireless service. Using this call you can get APN settings using tracker ID.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/apn_settings.md:8]: using tracker ID -> by tracker ID

File: docs/general/get-involved.md [hunk 3]
```diff
@@ -154,20 +154,40 @@ Method description.
...
+```
+
+### Response
 
 ```json
 { "success": true }
 ```
 
+!!! warning "Please note"
+    If the response or structure has comments it is necessary to write these comments separately in the form of a list below
```
Comment (fenik17) [inline docs/general/get-involved.md:189]: Dot misssed at the end.

File: docs/general/get-involved.md [hunk 4]
```diff
@@ -179,4 +199,4 @@ Special error codes.
 ````
 
 For real example see [/user](../backend-api/resources/commons/user/index.md) and
-[:octicons-file-code-24: source](https://raw.githubusercontent.com/SquareGPS/navixy-api/master/docs/backend-api/resources/commons/user/user.md).
\ No newline at end of file
+[:octicons-file-code-24: source](https://raw.githubusercontent.com/SquareGPS/navixy-api/master/docs/backend-api/resources/commons/user/user.md).
```
Comment (fenik17) [inline docs/general/get-involved.md:202]: Correct path: `..commons/user/index.md`

**Review by fenik17 — CHANGES_REQUESTED** [review]

**Comment by john9x** [comment]
Oh! BTW, why `abap` used for examples?

**Comment by fenik17** [comment]
@john9x Curiously enough `abap` well highlights console examples (wget). If you find a better option - welcome :)

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 5]
```diff
@@ -65,14 +82,22 @@ Marks all incoming chat messages as read for all or for given user trackers.
 
 | name | description | type | format |
 | :------ | :------ | :----- | :----- |
-| trackers | Optional array of Ids of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked. | array of int | [999199, 999919,...] |
+| trackers | Optional array of Ids of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked | array of int | [999199, 999919,...] |
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:85]: For why you remove a dot?

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 6]
```diff
@@ -162,89 +205,119 @@ Sends chat message to specified trackers
...
+
+=== "HTTP POST application/json
 
 ```abap
-$ curl -X POST '{{ extra.api_example_url }}/tracker/chat/broadcast/' \
--H 'Content-Type: application/json' \ 
--d '{"hash": "a6aa75587e5c59c32d347da438505fc3", "trackers": "[999199, 991999,...]", "message": "Hello World"}'
+$ curl -X POST '{{ extra.api_example_url }}/tracker/chat/broadcast' \
+    -H 'Content-Type: application/json' \ 
+    -d '{"hash": "a6aa75587e5c59c32d347da438505fc3", "trackers": "[999199, 991999,...]", "message": "Hello World"}'
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:215]: `,...` - remove it from example, please.

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 7]
```diff
@@ -162,89 +205,119 @@ Sends chat message to specified trackers
...
+    "value": {
         "101": "2016-02-29 00:23:00",
-        "122": "2017-02-28 00:23:00",
-        ...
+        "122": "2017-02-28 00:23:00"
     }
 }
 ```
 
+* value - map of trackers` IDs to date-times.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:280]: Wrong quotation mark.

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 8]
```diff
@@ -162,89 +205,119 @@ Sends chat message to specified trackers
...
     "success": true,
-    "value": { // map of trackers` IDs to counts
+    "value": {
       "1": 123,
       "2": 321
     }
 }
 ```
 
+* value - map of trackers` IDs to counts.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:319]: Wrong quotation mark.

File: docs/backend-api/resources/tracking/tracker/contact.md [hunk 9]
```diff
@@ -2,47 +2,62 @@
...
 ```
 
 #### response
 
 ```json
 {
     "success": true,
-    "contacts": [ <contact>, ... ] // all established contacts
-    "trackers": [ <tracker>, ... ] // normal trackers belonging to current user
+    "contacts": ["contact1","contact n"],
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/contact.md:37]: Missed whitespace between items.

File: docs/backend-api/resources/tracking/tracker/employee.md [hunk 10]
```diff
@@ -5,19 +5,39 @@ description: Assigning employee to tracker
 
 # Assigning employee to tracker
 
-## API actions
-
 API base path: `/tracker/employee`
 
+Allows you to assign employee("driver") to a device. Also, read who is on a vehicle now, hardware key and when, where was assigned.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/employee.md:10]: - Missed whitespace before `("driver")`.
- It is better not to use the pronoun "you".

File: docs/general/get-involved.md [hunk 11]
```diff
@@ -179,4 +199,4 @@ Special error codes.
 ````
 
 For real example see [/user](../backend-api/resources/commons/user/index.md) and
-[:octicons-file-code-24: source](https://raw.githubusercontent.com/SquareGPS/navixy-api/master/docs/backend-api/resources/commons/user/user.md).
\ No newline at end of file
+[:octicons-file-code-24: source](..commons/user/index.md).
```
Comment (fenik17) [inline docs/general/get-involved.md:202]: No... I did not write the full path in previous review. I only meant that at the end you need to change `used.md` to `index.md`. Please, check links before committing.

**Review by fenik17 — CHANGES_REQUESTED** [review]

File: docs/backend-api/resources/tracking/tracker/alarm_mode.md [hunk 12]
```diff
@@ -15,28 +16,38 @@ Gets the state of alarm mode of device.
...
 
 ```json
 {
  "success": true,
- "enabled": {boolean} // true if alarm mode is enabled
+ "enabled": true
 }
 ```
 
+* enabled - true if alarm mode enabled.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/alarm_mode.md:44]: Please, escape field names, entity names, and literals:

```
* `enabled` - `true` if alarm mode enabled.
```

File: docs/backend-api/resources/tracking/tracker/chat.md (hunk 5 above)
Comment (Astarsys) [reply docs/backend-api/resources/tracking/tracker/chat.md:85]: because I don't leave dots at the end of descriptions in tables.

Comment (Astarsys) [reply docs/backend-api/resources/tracking/tracker/chat.md:85]: but will add them from now

**Comment by john9x** [comment]
> 
> 
> @john9x Curiously enough `abap` well highlights console examples (wget). If you find a better option - welcome :)

`shell` looks good

изображение


Also, examples starts with `$` sign and button 'Copy to clipboard' copies all content of the block, so we need to remove `$` manually. Maybe we will write console examples without prefix?

File: docs/general/get-involved.md (hunk 11 above)
Comment (Astarsys) [reply docs/general/get-involved.md:202]: was confused because ..commons/user/index.md was eascaped with ` `

**Comment by Astarsys** [comment]
Yes, `$` can be deleted from our examples 
Also, I like `adap` highlighting more than `shell`

**Comment by fenik17** [comment]
I think it will be right if we remove `$` and change the highlighting style to `shell` for the console examples.

File: docs/backend-api/resources/tracking/tracker/engine_immobilizer.md [hunk 13]
```diff
@@ -1,37 +1,84 @@
...
 #### parameters
-* **tracker_id** - **int**. Id of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked.
+
+| name | description | type| format|
+| :------ | :------ | :----- | :------ |
+| tracker_id | Id of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked. | int | 123456 |
+
+#### examples
+
+=== "HTTP POST application/json
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/engine_immobilizer.md:23]: Missing closing quote. I suggest using `cURL` and `HTTP GET` for tab titles to make it more clear.

Comment (Astarsys) [reply docs/backend-api/resources/tracking/tracker/engine_immobilizer.md:23]: just to confirm 
1) we use `shell` for curl and `adap` for GET
2) we use the next names:
"HTTP POST application/json" -> "cURL"
and
"GET" -> "HTTP GET"?

File: docs/backend-api/resources/tracking/tracker/alarm_mode.md [hunk 14]
```diff
@@ -46,17 +55,23 @@ Changes the state of alarm mode of device. The device must be online.
...
+| enabled | True if alarm mode should be enabled. | boolean | true/false |
 
-#### example
+#### examples
 
-```abap
-$ curl -X POST '{{ extra.api_example_url }}/tracker/alarm_mode/set' \
--H 'Content-Type: application/json' \ 
--d '{"tracker_id": "999199", "enabled": "true", "hash": "a6aa75587e5c59c32d347da438505fc3"}'
+=== cURL
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/alarm_mode.md:63]: Tabs should have this syntax:

````
=== "cURL"

    ```shell
    curl -X POST '{{ extra.api_example_url }}/tracker/apn_settings/read' \
        -H 'Content-Type: application/json' \ 
        -d '{"tracker_id": "123456", "hash": "a6aa75587e5c59c32d347da438505fc3"}'
    ```

=== "HTTP GET"

    ```
    {{ extra.api_example_url }}/tracker/apn_settings/read?tracker_id=123456&hash=a6aa75587e5c59c32d347da438505fc3
    ```
````

See https://squidfunk.github.io/mkdocs-material/reference/content-tabs/

**Review by fenik17 — CHANGES_REQUESTED** [review]

File: docs/backend-api/resources/tracking/tracker/alarm_mode.md (hunk 14 above)
Comment (Astarsys) [reply docs/backend-api/resources/tracking/tracker/alarm_mode.md]: ready

File: docs/backend-api/resources/tracking/tracker/contact.md [hunk 15]
```diff
@@ -2,47 +2,62 @@
...
+    ```
 
 #### response
 
 ```json
 {
     "success": true,
-    "contacts": [ <contact>, ... ] // all established contacts
-    "trackers": [ <tracker>, ... ] // normal trackers belonging to current user
+    "contacts": ["contact1", "contact n"],
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/contact.md:37]: They are not strings, but objects.

File: docs/backend-api/resources/tracking/tracker/contact.md [hunk 16]
```diff
@@ -2,47 +2,62 @@
...
+    "trackers": ["tracker1", "tracker n"]
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/contact.md:38]: They are not strings, but objects.

File: docs/backend-api/resources/tracking/tracker/contact.md [hunk 17]
```diff
@@ -2,47 +2,62 @@
...
 
 ```json
 {
-    "user_id": 12059, //id of the user with which "contact" is established
+    "user_id": 12059,
     "first_name": "Adam",
     "middle_name": "James",
     "last_name": "Williams",
-    "trackers": [ <tracker>, ... ] //trackers belonging to "contact" which locations were shared with current user
+    "trackers": ["tracker1", "tracker n"]
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/contact.md:53]: They are not strings, but objects.

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 18]
```diff
@@ -13,49 +14,65 @@ Gets a list of chat messages.
...
+    {{ extra.api_example_url }}/tracker/chat/list?tracker_id=123456&hash=a6aa75587e5c59c32d347da438505fc3
+    ```
 
 #### response
 
 ```json
 {
     "success": true,
-    "list":[...] // [array of messages]
+    "list": ["message1", "message2"]
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:44]: They are not strings, but objects.

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 19]
```diff
@@ -114,137 +147,177 @@ $ curl -X POST '{{ extra.api_example_url }}/tracker/chat/mark_read/' \
...
+Sends chat message to a specified tracker.
 
 #### parameters
 
 | name | description | type | format |
 | :------ | :------ | :----- | :----- |
-| tracker_id | Id of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked | int | 999199 |
-| message | message text, not null, max size - 20000 | string | Hello World |
+| tracker_id | Id of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked. | int | 123456 |
+| message | message text, not null, max size - 20000. | string | Hello World |
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:161]: Capitalize the first word in description, please.

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 20]
```diff
@@ -114,137 +147,177 @@ $ curl -X POST '{{ extra.api_example_url }}/tracker/chat/mark_read/' \
...
-Sends chat message to specified trackers
+Sends chat message to specified trackers.
 
 #### parameters
 
 | name | description | type | format |
 | :------ | :------ | :----- | :----- |
-| trackers | array of Ids of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked. Max size - 300 | array of int | [999199, 999919,...] |
-| message | message text, not null, max size - 20000 | string | Hello World |
+| trackers | array of Ids of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked. Max size - 300. | array of int | [999199, 999919,...] |
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:205]: Capitalize the first word in description, please.

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 21]
```diff
@@ -114,137 +147,177 @@ $ curl -X POST '{{ extra.api_example_url }}/tracker/chat/mark_read/' \
...
+    "value": {
         "101": "2016-02-29 00:23:00",
-        "122": "2017-02-28 00:23:00",
-        ...
+        "122": "2017-02-28 00:23:00"
     }
 }
 ```
 
+* `value` - map of `trackers' IDs` to date-times.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:280]: Just "tracker IDs" - escaping is not needed.

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 22]
```diff
@@ -114,137 +147,177 @@ $ curl -X POST '{{ extra.api_example_url }}/tracker/chat/mark_read/' \
...
 
-* 217 – The list contains non-existent entities – if one of the specified trackers does not exist, is blocked or doesn't have required tariff features
-* 221 – Device limit exceeded (if device limit set for the user’s dealer has been exceeded)
+* 217 – The list contains non-existent entities – if one of the specified trackers does not exist, is blocked or doesn't have required tariff features.
+* 221 – Device limit exceeded (if device limit set for the user’s dealer has been exceeded).
 
 ### unread/count
 
-Gets count of user’s unread chat messages grouped by tracker id.
+Gets count of user’s unread chat messages grouped by `tracker id`.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:289]: Just "tracker ID" - escaping is not needed.

File: docs/backend-api/resources/tracking/tracker/chat.md [hunk 23]
```diff
@@ -114,137 +147,177 @@ $ curl -X POST '{{ extra.api_example_url }}/tracker/chat/mark_read/' \
...
     "success": true,
-    "value": { // map of trackers` IDs to counts
+    "value": {
       "1": 123,
       "2": 321
     }
 }
 ```
 
+* `value` - map of `trackers' IDs` to counts.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/chat.md:319]: Just "tracker IDs" - escaping is not needed.

**Review by fenik17 — CHANGES_REQUESTED** [review]

**Pull request merged**